src/ui/components/Transcript/TranscriptItem/FloatingTranscriptItem.tsx

```diff
@@ -1,43 +1,44 @@
-import React, { useState, useEffect } from "react";
+import React from "react";
 import { connect, ConnectedProps } from "react-redux";
 
 import CommentThread from "ui/components/Comments/TranscriptComments/CommentThread";
 import TranscriptItem from "./TranscriptItem";
-import { PendingComment, PauseItem, SourceLocation } from "ui/state/comments";
-import { ThreadFront } from "protocol/thread";
+import { FloatingItem } from "ui/state/comments";
 import { UIState } from "ui/state";
+import { selectors } from "ui/reducers";
 
 const { getFilenameFromURL } = require("devtools/client/debugger/src/utils/sources-tree/getURL");
 const { getTextAtLocation } = require("devtools/client/debugger/src/reducers/sources");
 const { getFormattedTime } = require("ui/utils/timeline");
+const { findClosestFunction } = require("devtools/client/debugger/src/utils/ast");
+const { getSymbols } = require("devtools/client/debugger/src/reducers/ast");
```

Comment:
in my initial patch, I made `findClosestFunction` a selector so that we could query it directly. Not a big deal tho